cis_v130/docs/cis_v130_1_1.md

```diff
@@ -0,0 +1,25 @@
+## Description
+
+Enable multi-factor authentication for all user credentials who have write access to Azure resources. These include roles like
+
+- Service Co-Administrators
+- Subscription Owners
+- Contributors
+
+Multi-factor authentication requires an individual to present a minimum of two separate forms of authentication before access is granted. Multi-factor authentication provides additional assurance that the individual attempting to gain access is who they claim to be. With multi-factor authentication, an attacker would need to compromise at least two different authentication mechanisms, increasing the difficulty of compromise and thus reducing the risk.
+
+**Note:** By default, multi-factor authentication is disabled for all users.
+
+## Remediation
+
+To find users in Azure enabled with MFA or not:
```

Comment:
These lines, along with ones like `To check and remediate:`, feel extra and don't add much. The CIS docs may have these, but we probably don't need them in our docs.